### PR TITLE
Task/COOKS-310: complete saga and update selectors

### PR DESCRIPTION
**Pull request by ryamase**
## Overview: ##
Complete new saga, update selectors, minor adjustments to chart section (e.g. instructions & new header)

## Related Jira tickets: ##

* [COOKS-123](https://jira.tacc.utexas.edu/browse/COOKS-123)

## Summary of Changes: ##

- New redux saga and reducer for getting statewide analytics charts
- Updated selectors per Juliana's design document
- Added header and updated instruction in the chart section

## Testing Steps: ##
1. Go to https://cep.test/protx/dash/analytics
2. Test the new dropdown menu for Projections and ensure that selecting each option ('Relative Risk' or 'Predicted Number of Cases') causes the intended chart to display.
3. Note any desired changes to chart header and/or instructions.

## UI Photos:
**Relative Risk Chart Selected**
Screen Shot 2022-10-19 at 12 22 29 PM

**Predicted Number of Cases Chart Selected**
Screen Shot 2022-10-19 at 12 23 32 PM

### CODE REVIEW

File: protx-client/src/components/ProTx/components/charts/AnalyticsStateDistribution.jsx [hunk 1]
```diff
@@ -4,32 +4,25 @@ import PropTypes from 'prop-types';
 import { LoadingSpinner } from '_common';
 import MainPlot from './MainPlot';
 import ChartInstructions from "./ChartInstructions";
+import './PlotDetails.css';
 
 
-function AnalyticsStateDistribution({geography}) {
+function AnalyticsStateDistribution({geography, analyticsType}) {
```
Comment (nathanfranklin) [inline protx-client/src/components/ProTx/components/charts/AnalyticsStateDistribution.jsx:10]: suggestion: add `analyticsType` to the propTypes.

[ we don't have any client-side linting for these things at the moment. we should tackle that next sprint]

File: protx-client/src/components/ProTx/components/dashboard/DisplaySelectors.jsx [hunk 2]
```diff
@@ -78,12 +78,14 @@ function DisplaySelectors({
   geography,
   maltreatmentTypes,
   observedFeature,
+  analyticsType,
```
Comment (nathanfranklin) [inline protx-client/src/components/ProTx/components/dashboard/DisplaySelectors.jsx:81]: add analyticsType and setAnalyticsType to proptypes. (probably as optional with default values of `null`)?

File: protx-client/src/components/ProTx/components/dashboard/DisplaySelectors.jsx [hunk 3]
```diff
@@ -98,6 +100,7 @@ function DisplaySelectors({
   const valueRadioBtn1 =
     mapType === 'maltreatment' ? 'rate_per_100k_under17' : 'count';
   const display = useSelector(state => state.protx.data.display);
+  const analyticsCategories = [{name: 'risk', display_text: 'Relative Risk'}, {name: 'pred_per_100k', display_text: 'Predicted Number of Cases'}]
```
Comment (nathanfranklin) [inline protx-client/src/components/ProTx/components/dashboard/DisplaySelectors.jsx:103]: reminder for us to ask Juliana about the "Predicted Number of Cases " string.  I wonder if we need the a "(per 100k)" added 🤷

**Review by nathanfranklin — CHANGES_REQUESTED** [review]
Left some minor suggestions (mostly as our listing is not active for client-side code, I just noticed two things about properties that we can alter before merging).  Otherwise, looks good to me 👍

**Comment by ryamase** [comment]
Good catch with the propTypes and defaultProps; I've made the suggested changes.

**Review by nathanfranklin — APPROVED** [review]
LGTM 👍

**Pull request merged**